### PR TITLE
Konstantinos / [FE Deriv.com squad] Update footer for deriv.com and eu.deriv.com

### PR DESCRIPTION
**Pull request by konstantinosG-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [x] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by habib-deriv — APPROVED** [review]
LGTM

File: src/store/region-context.tsx [hunk 1]
```diff
@@ -68,6 +70,8 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_eu = is_eu_location || isEuDomain()
         const is_non_eu = !is_eu
         const is_cpa_plan = cpa_plan_countries.includes(user_ip_country)
+        const first_two_countries = cpa_plan_countries.slice(0, 2)
+        const is_cpa_plan_no_bw = first_two_countries.includes(user_ip_country)
```
Comment (NikitK-deriv) [inline src/store/region-context.tsx:74]: You're already remove 'bw' from `cpa_plan_countries`. 
In this case `cpa_plan_countries` === `is_cpa_plan_no_bw`
Are we exactly need to rename and slice it?

**Review by NikitK-deriv — COMMENTED** [review]
Please resolve sonarlint code smell 🙏

File: src/store/region-context.tsx (hunk 1 above)
Comment (konstantinosG-deriv) [reply src/store/region-context.tsx:74]: Yeah true, forgot to remove it thanks!

**Review by habib-deriv — APPROVED** [review]

**Pull request merged**